### ISSUE TITLE
bower.io

### ISSUE THREAD

**Issue by scarlight** [jquery-form/form#447]
If you could, please update the latest version in bower as well. PS: Great plugin.

**Comment by johnpaulett** [comment]
Looks like bower.json is fine, perhaps it is the version numbers of the tags, needing to be in semver-style, e.g. 3.51.0 instead of 3.51?  

When I do `bower info jquery-form`, the only available versions are 3.46.0, 3.45.0, and 3.37.0 -- the only 3 semver tags I see in this repo. 

Related #451 & #402

**Comment by vjrabanelly** [comment]
It is indeed due to the version tag not following semver. @malsup Please use 3.52.0 for the next release. 

@johnpaulett You can specify "jquery-form": "3.51" in your bower.json, it will work, but it will send a weird warning and disappear if you use `bower prune`.

```
$ bower list
├─┬ jquery-form#3.51 incompatible with 3.51 (latest is 3.46.0)
│ └── jquery#2.1.3
```

**Comment by kevindb** [comment]
Probably resolved by 0dc9b4e
The new release has just been posted. I'll check back in a day or two to see if the package managers have picked up on the new version.

**Comment by kevindb** [comment]
Bower has been updated with [v4.1.0](https://github.com/jquery-form/form/releases/tag/v4.1.0)
